packages/core/src/html/SiteLinkManager.ts

```diff
@@ -69,4 +74,57 @@ export class SiteLinkManager {
     this._addToCollection(resourcePath, cwf);
     return 'Intralink collected to be validated later';
   }
+
+  /**
+   * Add sections that could be reached by intra-link with hash to this node to FilePathToHashesMap,
+   * The reachable sections include nodes with ids and headings.
+   *
+   * ForceWrite should only be called when processing heading node with the maintainHashesForInclude method.
```

Comment:
```suggestion
   * forceWrite should only be used when processing heading node.
```
Could you explain why?

Reply:
I think also the name `forceWrite` is not accurate. This also seems like a boolean instead of an override

Reply:
Hi, I have tried improving the code quality in a later commit; this parameter no longer exists.